### PR TITLE
fixed bug #21692: Different behaviour when clicking on file-link

### PR DESCRIPTION
**Pull request by alex40724**
See bug
https://www.ilias.de/mantis/view.php?id=21692

### CODE REVIEW

**Comment by alex40724** [comment]
@klees @Amstutz Please review

File: src/UI/Component/Link/Factory.php [hunk 1]
```diff
@@ -22,6 +22,9 @@
 	 *       1: >
 	 *          Standard links MUST be used if there is no good reason using
 	 *          another instance.
+	 *       2: >
+	 *          Links to ILIAS screens that contain the general ILIAS
+	 *          navigation MUST NOT be opened in a new viewport.
```
Comment (Amstutz) [inline src/UI/Component/Link/Factory.php:27]: +1

File: src/UI/templates/default/Link/tpl.standard.html [hunk 2]
```diff
@@ -1 +1 @@
-<a href="{HREF}">{LABEL}</a>
\ No newline at end of file
+<a href="{HREF}" <!-- BEGIN open_in_new_viewport --> target="_blank" <!-- END open_in_new_viewport -->>{LABEL}</a>
```
Comment (Amstutz) [inline src/UI/templates/default/Link/tpl.standard.html:1]: +1

File: tests/UI/Component/Link/LinkTest.php [hunk 3]
```diff
@@ -53,4 +53,19 @@ public function test_render_link() {
 
 		$this->assertHTMLEquals($expected_html, $html);
 	}
+
+	public function test_render_with_new_viewport() {
```
Comment (Amstutz) [inline tests/UI/Component/Link/LinkTest.php:57]: +1

**Review by Amstutz — APPROVED** [review]
Thanks for this enhancement of the link. I like the rule, the way you added this the the template and the new test.

From MPV this can be merged. Richard is aways for the rest of the week, but I guess he will be fine with this as well. If not, he might always open a new PR asking for some aspect to be changed.

Please check, if all Unit Tests pass. If they do you might merge this (I think it is better if you do the merging, since there is also some calendar code touched).

**Comment by matthiaskunkel** [comment]
Jour Fixe, 22 JAN 2018 : We highly appreciate this PR. Please merge it to 5.3 and trunk.

**Pull request merged**